test/integration/go.mod

```diff
@@ -1,3 +1,8 @@
 module github.com/GoogleCloudPlatform/terraform-google-secured-data-warehouse/test/integration
 
 go 1.16
+
+require (
+	github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test v0.0.0-20220419151121-2b888ce74c4d
```

Comment:
@renato-rudnicki we have new versions of these libraries: https://github.com/GoogleCloudPlatform/terraform-google-cloud-functions/blob/main/test/integration/go.mod#L5